requirements.dev.txt

```diff
@@ -0,0 +1,18 @@
+gitpython
```

Comment:
I already have all those packages in megalinter/setup.py , do we need to duplicate them here ? ( if yes no problem, I just don't know)